tests/installation/grub_test.pm

```diff
@@ -39,7 +39,9 @@ sub handle_installer_medium_bootup {
         assert_screen 'grub2';
     }
 
-    send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
+    # Layout of live is different from installation media
+    my $key = is_livecd ? 'down' : 'up';
```

Comment:
```suggestion
    my $key = is_livecd() ? 'down' : 'up';
```

Reply:
Updated accordingly. Thanks for your help!